<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Registration</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>33</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Città</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>39</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sesso</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Età</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_country">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>42</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cognome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_surname">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sex">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox_age">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>18</number>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_password">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_biography">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>239</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_signup">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrati</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Biografia</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_conn_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]Status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
